addons/hr_recruitment/wizard/applicant_send_mail.py

```diff
@@ -45,5 +56,24 @@ def action_send(self):
                 email_layout_xmlid='mail.mail_notification_light',
                 message_type='comment',
                 partner_ids=applicant.partner_id.ids,
-                subject=self.subject,
+                subject=subjects[applicant.id],
+                attachment_ids=self.attachment_ids.ids
             )
+
+    def action_save_as_template(self):
```

Comment:
@tde-banana-odoo asked to removed that part because:
- This will cause access rights issues
- The goal is to remove completely this feature in standard to push people using the related configuration menu and to avoid bloating the database with a shitload of unused templates